nucypher/cli/main.py

```diff
@@ -242,17 +254,71 @@ def ursula(click_config,
                                                      provider_uri=provider_uri,
                                                      poa=poa)
 
-        click.secho("Generated keyring {}".format(ursula_config.keyring_dir), fg='green')
-        click.secho("Saved configuration file {}".format(ursula_config.config_file_location), fg='green')
+        if not quiet:
+            click.secho("Generated keyring {}".format(ursula_config.keyring_dir), fg='green')
+            click.secho("Saved configuration file {}".format(ursula_config.config_file_location), fg='green')
+
+            # Give the use a suggestion as to what to do next...
+            how_to_run_message = "\nTo run an Ursula node from the default configuration filepath run: \n\n'{}'\n"
+            suggested_command = 'nucypher ursula run'
+            if config_root is not None:
+                config_file_location = os.path.join(config_root, config_file or UrsulaConfiguration.CONFIG_FILENAME)
+                suggested_command += ' --config-file {}'.format(config_file_location)
+            click.secho(how_to_run_message.format(suggested_command), fg='green')
+            return  # FIN
+
+        else:
+            click.secho("OK")
+
+    elif action == "destroy":
+        """Delete all configuration files from the disk"""
 
-        # Give the use a suggestion as to what to do next...
-        how_to_run_message = "\nTo run an Ursula node from the default configuration filepath run: \n\n'{}'\n"
-        suggested_command = 'nucypher ursula run'
-        if config_root is not None:
-            config_file_location = os.path.join(config_root, config_file or UrsulaConfiguration.CONFIG_FILENAME)
-            suggested_command += ' --config-file {}'.format(config_file_location)
-        click.secho(how_to_run_message.format(suggested_command), fg='green')
-        return  # FIN
+        if dev:
```

Comment:
With all these conditionals, maybe it's best to start pulling out some of this logic into functions that are called to reduce lines. What do you think?

Reply:
This code has been subject to that cycle sever times now, not that it cant be improved more; However, I actually think `cli.main` is pretty concise and readable.  

No less, Im curious what your suggestions might be.